Shopify/cli
# Remove flag requirement for log streaming

Removing the beta flag to GA log streaming, and generating all relevant documentation.

The documentation in this PR is all auto-generated.

### How to test your changes?

Unset `SHOPIFY_CLI_ENABLE_APP_LOG_POLLING`
Run shopify app logs, it should run without complaint
Run shopify app dev, it should stream logs
Run shopify app function replay, it should run without a complaint
Run shopify app --help, it should show the app logs command
Run shopify app function --help, it should show the replay command

Tophatted 🎩 ✅ 

### Checklist
